desc/geometry/surface.py

```diff
@@ -357,6 +361,200 @@ def from_near_axis(cls, aspect_ratio, elongation, mirror_ratio, axis_Z, NFP=1):
         surf = cls(R_lmn=R_lmn, Z_lmn=Z_lmn, modes_R=modes_R, modes_Z=modes_Z, NFP=NFP)
         return surf
 
+    @classmethod
+    def from_values(cls, coords, nodes, M, N, NFP, sym=True):
```

Comment:
Should the other PR be merged first? Or can we combine these two?

Reply:
I think other should be merged first

Reply:
So this method is from the other PR? Or is this a separate implementation?

Reply:
That other PR is a cherrypick from this PR, now that that one is merged to master, this PR is using the updated method implementation from the master branch